### PR TITLE
Change EU exit urls to brexit

### PR DESCRIPTION
**Pull request by eadpearce**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: conf/urls.py [hunk 1]
```diff
@@ -248,11 +248,21 @@
         euexit.views.InternationalContactFormView.as_view(),
         name='eu-exit-international-contact-form'
     ),
+    url(
+        r'^international/brexit/contact/$',
```
Comment (eadpearce) [inline conf/urls.py]: old urls will be kept until the url changes are in production for both international and domestic

File: conf/tests/test_redirects.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    ),
+]
+
+
+@pytest.mark.parametrize('url,expected', redirects)
+def test_redirects(url, expected, client):
+    response = client.get(url)
+
+    assert response.status_code == http.client.FOUND
+    if not expected.startswith('http') and not expected.startswith('/'):
```
Comment (MichaelWalker) [inline conf/tests/test_redirects.py]: What's this line for? It doesn't seem to ever be used by the parameters for this test?

Comment (eadpearce) [reply conf/tests/test_redirects.py]: no idea. i just copied it from the test that was in domestic repo

Comment (MichaelWalker) [reply conf/tests/test_redirects.py]: I think we can just delete it for this test.

Comment (eadpearce) [reply conf/tests/test_redirects.py]: oh wait i see how it's used now. i'll update the test to use that logic

**Pull request merged**